src/moveit_visual_tools.cpp

```diff
@@ -722,7 +733,32 @@ bool MoveItVisualTools::publishCollisionCuboid(const geometry_msgs::Pose& pose,
   if (!collision_obj.primitives[0].dimensions[shape_msgs::SolidPrimitive::BOX_Z])
     collision_obj.primitives[0].dimensions[shape_msgs::SolidPrimitive::BOX_Z] = rviz_visual_tools::SMALL_SCALE;
 
-  return processCollisionObjectMsg(collision_obj, color);
+  return collision_obj;
+}
+
+moveit_msgs::CollisionObject MoveItVisualTools::createCollisionObject(const Eigen::Isometry3d& pose, double width,
+                                                                      double depth, double height,
+                                                                      const std::string& name)
+{
+  geometry_msgs::Pose pose_msg = tf2::toMsg(pose);
```

Comment:
typically in *_visual_tools, we use the built-in converter helper function rather than tf2, could you use that? e.g. ``convertPose()``

same below

Reply:
done